### PR TITLE
dbeaver#19890 Add information about current Project to the tab tooltip

### PR DESCRIPTION
**Pull request by yieviezhang**

### CODE REVIEW

**Comment by E1izabeth** [comment]
Thank you for the contribution.
The information about project should be added not only to SQL editor tabs. We also have at least Data editor and Entity editors. Their tooltips should also be modified.

**Review by ShadelessFox — CHANGES_REQUESTED** [review]
Please remove unrelated changes.

File: plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java [hunk 1]
```diff
@@ -2191,7 +2191,8 @@ public String getTitleToolTip() {
             .append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_path, scriptPath))
             .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_connecton, dataSourceContainer.getName()))
             .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_type, dataSourceContainer.getDriver().getFullName()))
-            .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_url, dataSourceContainer.getConnectionConfiguration().getUrl()));
+            .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_url, dataSourceContainer.getConnectionConfiguration().getUrl()))
+            .append("\n").append(NLS.bind(SQLEditorMessages.sql_sql_editor_title_tooltip_project, dataSourceContainer.getFolder().getName()));
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java:2217]: A data source folder is not the same as a project it's contained in.

Use `DBPDataSourceContainer#getProject`

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java (hunk 1 above)
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java:2217]: dataSourceContainer.getFolder() may be null.

**Review by serge-rider — CHANGES_REQUESTED** [review]

**Comment by LonwoLonwo** [comment]
We are still waiting for other changes.

**Comment by ShadelessFox** [comment]
There were no updates for a long time. I'm closing this PR as it doesn't fix the requested changes.

**Pull request closed without merging**